.dockerignore

```diff
@@ -2,7 +2,6 @@
 **/.venv
 **/.classpath
 **/.dockerignore
-**/.env

```

Comment:
I usually make a .env.sample with empty value to let other dev know what to add. 

```
POSTGRES_USER=
POSTGRES_PASSWORD=
POSTGRES_DB=
POSTGRES_HOST=
POSTGRES_PORT=
```

Reply:
Then add a step in README.md telling developers to copy .env.sample to .env to fill in credentials.